File: blog/20110515/morphia_sample/src/main/scala/Book.scala
package fits.sample

import java.util.{List, ArrayList}
import com.google.code.morphia.annotations._
import org.bson.types.ObjectId

@Entity(value = "books", noClassnameStored = true)
class Book(@Property var title: String, @Property var isbn: String) {

	//デフォルトコンストラクタは必須
	def this() = this("", "")

	@Id
	var id: ObjectId = null

	@Embedded
	var comments: List[Comment] = new ArrayList[Comment]()

}

File: java/resteasy/resteasy_json_jackson_sample/src/main/java/fits/sample/model/Book.java
package fits.sample.model;

import java.util.ArrayList;
import java.util.List;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class Book {
	public String id;
	public String title;

	public List<Comment> comments = new ArrayList<>();

	public Book() {
	}

	public Book(String id, String title) {
		this.id = id;
		this.title = title;
	}
}